ecl/agentexec/agentexec.cpp

```diff
@@ -212,20 +221,25 @@ int CEclAgentExecutionServer::run()
 
 //---------------------------------------------------------------------------------
 
-typedef std::tuple<unsigned, const char *, const char *> ThreadCtx;
+typedef std::tuple<IJobQueueItem *, unsigned, const char *, const char *> ThreadCtx;
 
 // NB: WaitThread only used by if pool created (see CEclAgentExecutionServer ctor)
 class WaitThread : public CInterfaceOf<IPooledThread>
 {
+    CEclAgentExecutionServer &owner;
```

Comment:
Confusing to split the member variables - these should go with the others at the end.

Reply:
will move them.